sdk/cosmosdb/cosmos/src/CosmosDiagnostics.ts

```diff
@@ -118,6 +118,7 @@ export type GatewayStatistics = {
    * This is the activityId for request, made to server for fetching the requested resource. (As opposed to other potential meta data requests)
    */
   activityId?: string;
+  correlatedActivityId?: string;
```

Comment:
Shouldn't there be a doc-comment explaining what `correlatedActivityId` is and how it differs from `activityId`?

Reply:
Added comment